### PR TITLE
Adding dotnet quickstart for tracing

### PR DESCRIPTION
**Pull request by simonz130**
This quickstart is identical to tracing quickstart for other languages (ex. java).

### CODE REVIEW

File: content/quickstart/dotnet/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "C#"
+date: 2018-07-16T14:29:15-07:00
+draft: false
+class: "resized-logo"
+logo: /images/java-opencensus.png
```
Comment (odeke-em) [inline content/quickstart/dotnet/_index.md]: Please use the dotnet logo

File: content/quickstart/dotnet/_index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: "C#"
+date: 2018-07-16T14:29:15-07:00
```
Comment (odeke-em) [inline content/quickstart/dotnet/_index.md]: Please update this date or were you purposefully using this date to support listing C# after C++?
If so please instead use the `weight` attribute but really don't worry about that, Erlang is listed as the top so please use the date `date: 2019-01-07T00:01:21-07:00`

File: content/quickstart/dotnet/tracing.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
+---
+title: "Tracing"
+date: 2019-01-07T12:01:21-07:00
```
Comment (odeke-em) [inline content/quickstart/dotnet/tracing.md]: Please make this `date: 2019-01-07T00:01:21-07:00`

File: content/quickstart/dotnet/_index.md (hunk 2 above)
Comment (simonz130) [reply content/quickstart/dotnet/_index.md]: What date should I put here? Currently I put a random date.

Comment (odeke-em) [reply content/quickstart/dotnet/_index.md]: Please use `date: 2019-01-07T00:01:21-07:00` and I don't know if you saw this but I had commented with it before :)
screen shot 2019-01-08 at 6 43 43 am

File: content/quickstart/dotnet/tracing.md (hunk 3 above)
Comment (simonz130) [reply content/quickstart/dotnet/tracing.md]: done

File: content/quickstart/dotnet/_index.md (hunk 1 above)
Comment (simonz130) [reply content/quickstart/dotnet/_index.md]: ok

File: content/quickstart/dotnet/_index.md (hunk 2 above)
Comment (simonz130) [reply content/quickstart/dotnet/_index.md]: Thanks. I applied your date. Just curious - is there any rule on how the date should be set here in general?

Comment (odeke-em) [reply content/quickstart/dotnet/_index.md]: Just the current date for which you made a change.

**Comment by odeke-em** [comment]
Cool, thank you @simonz130 for the changes LGTM!

**Comment by simonz130** [comment]
Wow, that was fast response 👍 

Thanks!

**Pull request merged**